### PR TITLE
Remove FLoC handling

### PR DESCRIPTION
**Pull request by furudean**
[FLoC was canned by WICG in favor of the Topics API](https://github.com/WICG/floc#replaced-by-the-topics-api). I couldn't find any information online if it was removed from chrome yet (the only browser where it was ever implemented) but it definitely looks like its on its way out.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by mrkishi** [comment]
FLoC never made it past origin trials and it's indeed not active in any browsers anymore.

Having said that, origin trials for its successor have already begun and [the FLoC opt-out header is still accepted in the current Topics API draft](https://github.com/patcg-individual-drafts/topics#:~:text=Note%3A%20The%20old%20Permissions%2DPolicy%3A%20interest%2Dcohort%3D()%20from%20FLoC%20will%20also%20forbid%20topic%20calculation.).

We should probably add a Topics API opt-out option when removing FLoC's, but since it's still in flux and the deprecated option is still valid, I wouldn't oppose postponing any action until the spec moves forward.

**Comment by furudean** [comment]
I see, maybe we should change this to `Permissions-Policy: browsing-topics=()` instead to stay up to date? Or we can wait and see if topics also get scrapped... lol.

Though at this point we are definitely asking a different question--are topics as bad as FLoC, and should Svelte Kit do something about it by default?

**Comment by mrkishi** [comment]
> are topics as bad as FLoC

Getting your hands wet might not be as bad as drowning in water, but I wouldn't want it 24/7 either. I expect most on the team to share the uneasiness towards the new iteration as well.

If you _want_ to update the implementation and docs to get it in sync with Topics API instead of FLoC, I believe you'll find no objections. But I don't speak for everyone so you might want to wait for other opinions.

**Comment by mrkishi** [comment]
> We should probably add a Topics API opt-out option when removing FLoC's

I believe this would still be valuable even if privacy experts ultimately decide the Topics API isn't that harmful—it's just that if it turns out it _is_, then it should be enabled by default.

**Comment by Rich-Harris** [comment]
If we don't want to disable it by default then the opt-out would just be this in `hooks.js`, no?

```js
export async function handle({ event, resolve }) {
  const response = await resolve(event);
  response.headers.set('Permissions-Policy', 'browsing-topics=()');
  return response;
}
```

I'd argue we only want config for it if it's something we want to enable by default, which probably _does_ depend on how harmful it is.

File: .changeset/friendly-eggs-cry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+remove floc
```
Comment (Rich-Harris) [inline .changeset/friendly-eggs-cry.md:5]:
```suggestion
remove FLoC protection, now that we vanquished Google
```

**Comment by Rich-Harris** [comment]
thank you!

**Pull request merged**